integration.requirements

```diff
@@ -1 +1,2 @@
 molecule[docker]
+selinux
```

Comment:
:+1: for renaming. I was just thinking it would indeed be nice to rename `requirements-dev.txt` to `integration.requirements`, considering that _sanity_, _units_ and _integration_ are all development requirements.